get-involved.php

```diff
@@ -112,38 +94,74 @@
-
+        </div>
+        <div class="whole">
+            <h2>Our Progress for The Loki Alpha1 Milestone:</h2>
+            <div class="charts">
+                <div class="barchart-ctn">
+                    <canvas width="750" height="400"></canvas>
+                </div>
+                <div class="doughnuts-ctn">
+                    <div id="date"></div>
+                    <div class="doughnut fixed">
+                        <canvas id="fixed-chart" width="90" height="90"></canvas>
+                        <div class="doughnut-label">
+                            <span class="doughnut-count">0</span><br>
+                            Fixed
+                        </div>
+                    </div>
+                    <div class="doughnut in-progress">
+                        <canvas id="in-progress-chart" width="90" height="90"></canvas>
+                        <div class="doughnut-label">
+                            <span class="doughnut-count">0</span><br>
+                            In Progress
+                        </div>
+                    </div>
+                    <div class="doughnut created">
+                        <canvas id="created-chart" width="90" height="90"></canvas>
+                        <div class="doughnut-label">
+                            <span class="doughnut-count">0</span><br>
+                            Unfixed
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="whole">
             <div class="actions">
                 <a class="button flat suggested-action" href="https://code.launchpad.net/~elementary-pantheon" target="_blank">Browse Our Desktop Code</a>
                 <a class="button flat" href="https://code.launchpad.net/~elementary-apps" target="_blank">Browse Our Apps' Code</a>
-                <a class="button flat" href="https://bugs.launchpad.net/elementary" target="_blank">See Our Open Bug Reports</a>
+                <a class="button flat" href="https://bugs.launchpad.net/elementary/+bugs?field.tag=bitesize" target="_blank">Bitesized Bugs</a>
+                <a class="button flat" href="https://bugs.launchpad.net/elementary/+bugs?field.searchtext=&orderby=-importance&field.status%3Alist=CONFIRMED&field.status%3Alist=TRIAGED&assignee_option=any&field.assignee=&field.bug_reporter=&field.bug_commenter=&field.subscriber=&field.structural_subscriber=&field.tag=bounty&field.tags_combinator=ANY&field.has_cve.used=&field.omit_dupes.used=&field.omit_dupes=on&field.affects_me.used=&field.has_patch.used=&field.has_branches.used=&field.has_branches=on&field.has_no_branches.used=&field.has_no_branches=on&field.has_blueprints.used=&field.has_blueprints=on&field.has_no_blueprints.used=&field.has_no_blueprints=on&search=Search">Bountied Bugs</a>
```

Comment:
Why not use something shorter?
https://bugs.launchpad.net/elementary/+bugs?orderby=-importance&field.status%3Alist=CONFIRMED&field.status%3Alist=TRIAGED&field.tag=bounty&field.omit_dupes=on&field.has_branches=on&field.has_no_branches=on&field.has_blueprints=on&field.has_no_blueprints=on&search=Search